controller.go

```diff
@@ -96,21 +111,37 @@ func (c *Controller) Process(m *messaging.Message) [][]byte {
 
 		resp := c.bindACK(m.Source, params)
 		return CommandResponse{resp}
-	case "bitcoind":
-		var params BitcoindRPCParams
+	case "createwallet":
+		var params CreateWalletRPCParams
 		if err := json.Unmarshal(req.Args, &params); err != nil {
-			return CommandResponse{ErrorResponse(fmt.Errorf("bad request for bitcoind. error: %s", err.Error()))}
+			return CommandResponse{ErrorResponse(fmt.Errorf("bad request for createwallet. error: %s", err.Error()))}
 		}
 
-		resp := c.bitcoinRPC(m.Source, params)
+		resp := c.createWallet(params.Descriptor)
 		return CommandResponse{resp}
-	case "createwallet":
-		var params CreateWalletRPCParams
+	case "setmember":
```

Comment:
Not sure if it is a good convention to use command like this. Could you update the `bind_ack` to `bindack` if it is fine to do this way?

For me, I know `createwallet` so I do not feel weird. But for this two new command and later one, do we still use in this way?

Reply:
I was affected by the naming of Bitcoin RPC. I also like the snake case because of its readability. I would change them.